package.json

```diff
@@ -7,8 +7,15 @@
     "test": "tests"
   },
   "scripts": {
+    "setup": "npm run build",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "theme": "cd ./web/themes/custom/karaoke && yarn start"
+    "theme-build": "cd ./web/themes/custom/kitchen_karaoke && npm install && npm run build",
+    "theme": "cd ./web/themes/custom/kitchen_karaoke && npm run develop",
+    "build": "lando start && npm run get-db && lando composer install && lando drush updb -y && npm run theme-build && lando drush cr && lando drush uli ",
```

Comment:
Need to update the lando tooling to set the uri so uli will go to the correct address.